src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java

```diff
@@ -24,14 +24,21 @@
     @JsonProperty("scanOCRData")
     public final List<CcdCollectionElement<CcdKeyValue>> ocrData;
 
+    public final List<CcdCollectionElement<String>> ocrDataValidationWarnings;
+
+    // Yes/No field indicating if there are warnings to show
+    public final String hasWarnings;
+
     public ExceptionRecord(
         String classification,
         String poBox,
         String jurisdiction,
         LocalDateTime deliveryDate,
         LocalDateTime openingDate,
         List<CcdCollectionElement<ScannedDocument>> scannedDocuments,
-        List<CcdCollectionElement<CcdKeyValue>> ocrData
+        List<CcdCollectionElement<CcdKeyValue>> ocrData,
+        List<CcdCollectionElement<String>> ocrDataValidationWarnings,
+        String hasWarnings
```

Comment:
This can lead to creating ER with warning and this field being set to "No".
Or even "Hello".
Perhaps `hasWarnings` could be a getter?

Reply:
Tests prevent this scenario. I made it this way so that the model would remain without logic.

Reply:
should we rename hasWarnings to warningsIndicator or something similar as it is not a boolean type ?

Reply:
We should treat it like a boolean, though. This is a YesOrNo field in CCD, which corresponds to boolean. Unfortunately, CCD doesn't support boolean in its normal form.

Reply:
🤦‍♂